ssp

```diff
@@ -1735,7 +1735,8 @@ sub print_cpanel_info {
         '56' => { expires => 1509408001, text => 'October 31, 2017' },
         '58' => { expires => 1501459201, text => 'July 31, 2017' },
         '60' => { expires => 1509408001, text => 'October 31, 2017' },
-        '62' => { expires => 1522454401, text => 'March 31, 2018' }
+        '62' => { expires => 1522454401, text => 'March 31, 2018' },
+        '64' => { expires => 1505970000, text => 'September 21, 2017' }
```

Comment:
So good practice is to _always_ use the trailing comma when doing defines in lists/arrays/hashes
**Example:**
```
my %hash = {
    'expire1' => 'funk1',
    'expire2' => 'funk2',
    'expire3' => 'funk3',
);
```
The worry about that trailing definition simply doesn't need to negate the presence of the comma.  It can remain there so it's good practice to leave them there so we don't have to constantly jog lines because of that single comma being added/removed etc

Reply:
Thanks for the confirmation, I will be sure to keep that in mind in the future.